pkg/kademlia/kademlia.go

```diff
@@ -120,7 +124,119 @@ func (k *Kademlia) GetRoutingTable(ctx context.Context) (dht.RoutingTable, error
 // Bootstrap contacts one of a set of pre defined trusted nodes on the network and
 // begins populating the local Kademlia node
 func (k *Kademlia) Bootstrap(ctx context.Context) error {
-	return k.dht.Bootstrap()
+	if len(k.bootstrapNodes) == 0 {
+		return BootstrapErr.New("no bootstrap nodes provided")
+	}
+	// psuedo randomly select a node from the slice of bootstrap nodes
+	// TODO(coyle): should we instead bootstrap against all the nodes ?
```

Comment:
yeah, probably more robust to add all of your bootstrap nodes to the routing table, then do a vanilla k.Lookup for your own id (allowing k.Lookup to internally call k.nodeClient.Lookup iteratively). the s/kad stuff we add later will ensure that each bootstrap node is adequately considered